l10n_fi_invoice/models/account_journal.py

```diff
@@ -0,0 +1,14 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+# Copyright (C) Avoin.Systems 2020
+
+from odoo import models, fields
+
+
+class AccountJournal(models.Model):
+    _inherit = 'account.journal'
+
+    include_on_invoice = fields.Boolean(
+        "Include on Invoice",
+        help="Include this bank account in Finnish invoices generated for the associated company",
+        default=True
```

Comment:
Please change to `default=False` since this might have unexpected consequences, ie. when someone adds a new savings account and doesn't notice this flag.

Also add a mention in the `static/description/index.html` file that to make this module work properly, you must go to Journal settings and select "Include on Invoice" for each account that you want to show on the invoice. Also worth mentioning in the README.

Reply:
Forget what I just said. I think it's best to just remove this completely and go with the Odoo way of just using `accout.move.invoice_partner_bank_id`.

Reply:
`include_on_invoice` field is gone from this module